### DISCUSSION TITLE
mark WHOLE lines that are NOT on the right side

### DISCUSSION THREAD

**Discussion by PRhodan** [WinMerge/winmerge#595] [Q&A] [answered]
hello,
I just want to mark WHOLE lines that are NOT on the right side and then copy just those to the right.
Somehow everything is colorful and I can't really find anything.
Greetings

**Comment by sdottaka** [comment]
If the background of all lines is colored, EOL in the left and right files may be different. (e.g. CR+LF vs LF)

If you show us a screenshot of WinMerge in that state, we may be able to figure out the cause.

**Comment by PRhodan** [comment, marked as answer]
THX, my question is solved.
hello,
I was probably looking for a very specific application.
I have now even found a 100% for me.
WinMerge may be good, but too powerful for me. I was looking for something very simple.
greeting
